bin/initOSProjects.sh

```diff
@@ -35,3 +35,21 @@ for project in ${PROJECT_NAMESPACE}-${DEV} ${PROJECT_NAMESPACE}-${TEST} ${PROJEC
   exitOnError
 
 done
+
+#look through the tools env for artifactory-creds
```

Comment:
Move this initialization block and place it under the script import section (place it between what is now line 22 and 23).

Reply:
moved to top